meshtastic/config.proto

```diff
@@ -131,7 +131,7 @@ message Config {
     /*
      * Sets the role of node
      */
-    Role role = 1;
+    optional Role role = 1;
```

Comment:
I'm not totally sure this or the other in this file are worth it -- they do both have "real 0" values, but I guess neither makes that much sense to have unset, so we don't really need to distinguish. Unsure.